### PR TITLE
Add links for all EKF2 doc parameters.

### PR DESCRIPTION
**Pull request by hamishwillee**
@priseborough , can you please review this doc to confirm that it is still accurate.

In addition, there are several specific issues (from #223)
- The Drag Specific Forces chapter mentions EKF2_AID_MASK bit 4, but that is used for "4: vision yaw fusion"
- EKF2_BCOEF_X and EKF2_BCOEF_Y do not exist.
- At the end it says " \(insert example plots showing loss of GPS data here\)" - do you have those images?

### CODE REVIEW

File: en/tutorials/tuning_the_ecl_ekf.md [hunk 1]
```diff
@@ -42,59 +42,59 @@ Three axis body fixed magnetometer data \(or external vision system pose data\)
...
+Multi-rotor platforms can take advantage of the relationship between airspeed and drag force along the X and Y body axes to estimate North/East components of wind velocity. This is enabled by setting bit position 5 in the [EKF2_AID_MASK](../advanced/parameter_reference.md#EKF2_AID_MASK) parameter to true. The relationship between airspeed and specific force \(IMU acceleration\) along the X and Y body axes is controlled by the the [EKF2_BCOEF_X](../advanced/parameter_reference.md#EKF2_BCOEF_X) and [EKF2_BCOEF_Y](../advanced/parameter_reference.md#EKF2_BCOEF_Y) parameters which set the ballistic coefficients for flight in the X and Y directions respectively.. The amount of specific force observation noise is set by the [EKF2_DRAG_NOISE](../advanced/parameter_reference.md#EKF2_DRAG_NOISE) parameter.
```
Comment (hamishwillee) [inline en/tutorials/tuning_the_ecl_ekf.md:77]: EKF2_BCOEF_X and EKF2_BCOEF_Y are not listed in parameters. I checked all other links in this doc and they do work.

File: en/tutorials/tuning_the_ecl_ekf.md [hunk 2]
```diff
@@ -323,15 +327,17 @@ Poor GPS accuracy is normally accompanied by a rise in the reported velocity err
...
+If we also look at the GPS horizontal velocity innovations and innovation variances, we can see the large spike in North velocity innovation that accompanies this GPS 'glitch' event.
+
 ![](../../assets/ecl/gps_glitch_-_velocity_innovations.png)
 
 ### Determination of GPS Data Loss
 
-Loss of GPS data will be shown by the velocity and position innvoation test ratios 'flat-lining'. If this occurs, check the oher GPS status data in vehicle\_gps\_position for further information.
+Loss of GPS data will be shown by the velocity and position innovation test ratios 'flat-lining'. If this occurs, check the other GPS status data in `vehicle_gps_position` for further information.
 
 \(insert example plots showing loss of GPS data here\)
```
Comment (hamishwillee) [inline en/tutorials/tuning_the_ecl_ekf.md]: These images are missing

File: en/tutorials/tuning_the_ecl_ekf.md (hunk 1 above)
Comment (priseborough) [reply en/tutorials/tuning_the_ecl_ekf.md:77]: I don't know why. They appear here: https://github.com/PX4/Firmware/blob/master/src/modules/ekf2/ekf2_params.c#L1082 and here: https://github.com/PX4/Firmware/blob/master/src/modules/ekf2/ekf2_params.c#L1094

File: en/tutorials/tuning_the_ecl_ekf.md (hunk 2 above)
Comment (priseborough) [reply en/tutorials/tuning_the_ecl_ekf.md]: I haven't yet received logs where GPS data was lost which is different to a temporary glitch. I will generate data using SITL.

**Comment by priseborough** [comment]
The following plot shows the NED GPS velocity innovations  ekf2_innovations_0.vel_pos_innov[0 ... 2], the GPS NE position innovations ekf2_innovations_0.vel_pos_innov[3 ... 4] and and the Baro vertical position innovation ekf2_innovations_0.vel_pos_innov[5] generated from a simulated VTOL flight using SITL Gazebo.

The simulated GPS was made to lose lock at 73 seconds. Note the NED velocity innovations and NE position innovations 'flat-line' after GPs is lost. Note that after 10 seconds without GPS data, the EKF reverts back to a static position mode using the last known position and the NE position innovations start to change again.

loss of gps lock

File: en/tutorials/tuning_the_ecl_ekf.md (hunk 1 above)
Comment (hamishwillee) [reply en/tutorials/tuning_the_ecl_ekf.md:77]: Thanks. Probably the params got out of sync. I will check as soon as this is fixed: https://github.com/PX4/Firmware/issues/8122

**Comment by hamishwillee** [comment]
Thanks @priseborough . I've added the graph and notes to the end. Will see what is happening about the parameters and then merge.

**Review by priseborough — APPROVED** [review]

**Comment by hamishwillee** [comment]
FYI, the links will be fixed after https://github.com/PX4/Firmware/issues/8122 is fixed and I regenerate the parameters.

**Comment by hamishwillee** [comment]
I'm merging this. The links may not work yet, but it is worth getting the other updates in.

**Comment by hamishwillee** [comment]
FYI, EKF2_BCOEF_X and EKF2_BCOEF_Y are now properly linked.

**Pull request merged**